### PR TITLE
reindexed all categories to allow for future expansion

### PR DESCRIPTION
**Pull request by GrahamTheDev**

### CODE REVIEW

File: docs/community.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 9
+sidebar_position: 100
```
Comment (FrancescoXX) [inline docs/community.md:2]: why 100 and not 90 ?

**Review by FrancescoXX — COMMENTED** [review]
please check the comment

File: docs/community.md (hunk 1 above)
Comment (FrancescoXX) [reply docs/community.md:2]: @GrahamTheDev

Comment (GrahamTheDev) [reply docs/community.md:2]: There were duplicate indexes so I reindexed accordingly, so 5 became 60, 6 became 70 etc.

**Pull request merged**